### ISSUE TITLE
Regarding flair/ner-english-ontonotes model fine tuning

### ISSUE THREAD

**Issue by pratikchhapolika** [flairNLP/flair#2914]
I thought of `Fine-Tuning` the model `flair/ner-english-ontonotes` here `https://huggingface.co/flair/ner-english-ontonotes`.

I couldn't find the following files in `Files and Verions` https://huggingface.co/flair/ner-english-ontonotes/tree/main

- tokenizer_config.josn
- config.json
- some more tokens related files

In all the Hugging face models, we have these files. 

**Am I missing something?**

If these are the only files, is there any code in `Google colab` which I can use to fine-tune this model based on my data and labels.

My data looks like this:

```
{text: "Texas USA TX DRIVER LICENSE B Lamma 4d DL 12345678 9 Class AM to Iss 04/05/21 Ab Exp 07130/2021 DOB 03/03/2021 
1 B 2 GERALD 8 2120 OLD MAIN STREET ANYTOWN TX 123456-0000 073076 12 Restrictions A 9a End P 16 Hgt 5'-04" 15 Sex F 18 Eyes 
BLU 5 DD 12345678900000000000 Gerald B",
tags:

 [{'start': 278, 'end': 293, 'label': 'PERSON_NAME', 'ngram': 'Gerald B'}, 
       {'start': 131, 'end': 137, 'label': 'FIRST_NAME', 'ngram': 'GERALD'}, 
       {'start': 118, 'end': 130, 'label': 'LAST_NAME', 'ngram': 'B'}, 
       {'start': 76, 'end': 84, 'label': 'ISSUE_DATE', 'ngram': '04/05/21'}, 
       {'start': 92, 'end': 102, 'label': 'EXPIRY_DATE', 'ngram': '07130/2021'}, 
       {'start': 107, 'end': 117, 'label': 'DATE_OF_BIRTH', 'ngram': '03/03/2021'}, 
       {'start': 49, 'end': 57, 'label': 'DRIVER_LICENSE_NUMBER', 'ngram': '12345678'}, 
       {'start': 140, 'end': 182, 'label': 'ADDRESS', 'ngram': '2120 OLD MAIN STREET ANYTOWN TX 123456-0000'}]}

```

**Comment by helpmefindaname** [comment]
> Am I missing something?

I think you are missing, that Flair creates Flair-models and not hugging face models. You can see how to use them by clicking on the `</> Use in Flair` button. 

To finetune models, you can select the fitting [Tutorial](https://github.com/flairNLP/flair#tutorials)

**Comment by pratikchhapolika** [comment]
> > Am I missing something?
> 
> I think you are missing, that Flair creates Flair-models and not hugging face models. You can see how to use them by clicking on the `</> Use in Flair` button on the huggingface hub.
> 
> To fine tune models, you can select the fitting [Tutorial](https://github.com/flairNLP/flair#tutorials)

Thank you for your response. I will definitely try to fine-tune using this tutorial. If I get stuck, I will ping for help. Meanwhile, can I keep this open?

**Comment by irg1008** [comment]
I have seen flair models working with HF inference API, and for me is working when using flair locally.

Can we not use the flair models with the inference API? An if we can, how? I having some trouble finding info on this subject

I have tried the tutorials above but I don't know the one you are referring to
